addons/survey/static/src/js/survey_form.js

```diff
@@ -726,7 +726,11 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend(SurveyPreloa
                         var subQuestionsIds = $questionWrapper.find('table').data('subQuestions');
                         subQuestionsIds.forEach(function (id) {
                             if (!((questionId + '_' + id) in data)) {
-                                errors[questionId] = constrErrorMsg;
+                                if (errors[questionId]){
+                                    errors[questionId]['row_ids'].push(id);
```

Comment:
In JS we prefer to use the shorter dot notation to access object properties
```suggestion
                                    errors[questionId].row_ids.push(id);
```